### PR TITLE
adaptive UI and removed unnecessary API endpoint

### PR DESCRIPTION
**Pull request by n4-t3**
fixed few adaptive UI issues and merged 2 api endpoints into one since they are kind of duplicated, from this comment https://github.com/kixlab/kuiz/pull/65#discussion_r1161263454

### CODE REVIEW

File: src/components/basic/Table.tsx [hunk 1]
```diff
@@ -33,6 +33,15 @@ export const Table = View<Props>(({ headers, rows, onClickRow, ...props }) => {
 
 const Container = styled.table`
   width: 100%;
+  @media (max-width: 590px) {
+    font-size: 10px;
+    margin-left: 0px;
+    margin-right: 0px;
+    padding: 0;
+  }
+  @media (max-width: 400px) {
```
Comment (jhw123) [inline src/components/basic/Table.tsx:42]: can you define these values as UI constants with meaningful names?

File: src/pages/class/[cid]/index.tsx [hunk 2]
```diff
@@ -180,6 +180,12 @@ const InformationContainer = styled.div`
   grid-template-columns: auto 1fr auto;
   gap: 40px;
   box-shadow: 0px 0px 16px rgba(40, 40, 40, 0.12);
+  @media (max-width: 570px) {
```
Comment (jhw123) [inline src/pages/class/[cid]/index.tsx:183]: change this too.

Comment (jhw123) [reply src/pages/class/[cid]/index.tsx:183]: could you add a screenshot of how this changes the view?

**Review by jhw123 — CHANGES_REQUESTED** [review]

File: src/components/basic/Table.tsx (hunk 1 above)
Comment (n4-t3) [reply src/components/basic/Table.tsx]: > can you define these values as UI constants with meaningful names?

Nice catch! updated it here https://github.com/kixlab/kuiz/pull/82/commits/a57bc9ea68f6c46b4a6fd4f4d306b6a7ef5b8941 :)

File: src/pages/class/[cid]/index.tsx (hunk 2 above)
Comment (n4-t3) [reply src/pages/class/[cid]/index.tsx]: > change this too.

Updated here https://github.com/kixlab/kuiz/pull/82/commits/a57bc9ea68f6c46b4a6fd4f4d306b6a7ef5b8941 :)

> could you add a screenshot of how this changes the view?

Here are the before and after screenshots :)

before


after

**Review by jhw123 — APPROVED** [review]

**Pull request merged**